jenkins.el

```diff
@@ -416,7 +421,10 @@
   "Call jenkins build JOBNAME function."
   (let ((url-request-extra-headers (jenkins--get-auth-headers))
         (url-request-method "POST")
-        (build-url (format "%sjob/%s/build" (get-jenkins-url) jobname)))
+        (build-url (format "%sjob/%s/build?%s"
```

Comment:
that '?' always will be in the end. If can can get rid of it when jenkins-build-parameter is not we can merge. thank you!

Reply:
I missed the notification. Sorry for long time no response. Please take a look again.